lib/structures/Interaction.js

```diff
@@ -0,0 +1,328 @@
+"use strict";
+
+const Base = require("./Base");
+const Message = require("./Message");
+const Member = require("./Member");
+const User = require("./User");
+const Constants = require("../Constants");
+
+/**
+* Represents an interaction
+* @prop {String} applicationID The id of the interaction's application
+* @prop {String?} channelID The interaction token
```

Comment:
```suggestion
* @prop {String?} channelID The ID of the channel in which the interaction was created
```
Sounds like a typo to me.